### ISSUE TITLE
示例代码疑似不完整，输出数据无法使用

### ISSUE THREAD

**Issue by newnewnew666** [Kwai-Keye/Keye#23]
from transformers import AutoModel, AutoProcessor
from keye_vl_utils import process_vision_info

# default: Load the model on the available device(s)
model_path = "Kwai-Keye/Keye-VL-8B-Preview"
device = "cuda"
model = AutoModel.from_pretrained(
    model_path, torch_dtype="auto", device_map=device, attn_implementation="flash_attention_2", trust_remote_code=True,
)

# You can set the maximum tokens for a video through the environment variable VIDEO_MAX_PIXELS
# based on the maximum tokens that the model can accept.
# export VIDEO_MAX_PIXELS = 32000 * 28 * 28 * 0.9

# You can directly insert a local file path, a URL, or a base64-encoded image into the position where you want in the text.
messages = [
    [{"role": "user",
      "content": [{"type": "video", "video": r"D:\Resource\Aworkspace\PythonSpace\NarratoAI-DontPushGithub\NarratoAI\storage\origin_videos\算死草-1.mp4"}, {"type": "text", "text": "提取视频中的字幕"}]}]
]

processor = AutoProcessor.from_pretrained(model_path, trust_remote_code=True)
# model = AutoModel.from_pretrained(model_path, torch_dtype="auto", device_map="cuda", trust_remote_code=True)
text = processor.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
print(f"text:{text}")
images, videos, video_kwargs = process_vision_info(messages, return_video_kwargs=True)
inputs = processor(text=text, images=images, videos=videos, padding=True, return_tensors="pt", **video_kwargs).to(device)
print(f"inputs:{inputs}")
generated_ids = model.generate(**inputs)
print(f"generated_ids:{generated_ids}")

输出

"D:\Program Files\Python-3.10.11\python.exe" D:\Resource\Aworkspace\PythonSpace\NarratoAI-DontPushGithub\NarratoAI\test\test_keye.py 
Loading checkpoint shards: 100%|██████████| 4/4 [00:07<00:00,  1.80s/it]
text:['<|im_start|>user\n<|vision_start|><|video_pad|><|vision_end|>提取视频中的字幕<|im_end|>\n<|im_start|>assistant\n']
keye-vl-utils using torchvision to read video.
inputs:{'input_ids': tensor([[151644,    872,    198,  ..., 151644,  77091,    198]],
       device='cuda:0'), 'attention_mask': tensor([[1, 1, 1,  ..., 1, 1, 1]], device='cuda:0'), 'pixel_values_videos': tensor([[[[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]]],

        [[[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]]],

        [[[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000]]],

        ...,

        [[[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9059, -0.9059, -0.9059,  ..., -0.9059, -0.9059, -0.9059],
          [-0.7804, -0.7804, -0.7804,  ..., -0.7804, -0.7804, -0.7804],
          [-0.7961, -0.7961, -0.7961,  ..., -0.7961, -0.7961, -0.7961]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9451, -0.9451, -0.9451,  ..., -0.9451, -0.9451, -0.9451],
          [-0.8196, -0.8196, -0.8196,  ..., -0.8196, -0.8196, -0.8196],
          [-0.8353, -0.8353, -0.8353,  ..., -0.8353, -0.8353, -0.8353]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9529, -0.9529, -0.9529,  ..., -0.9529, -0.9529, -0.9529],
          [-0.8275, -0.8275, -0.8275,  ..., -0.8275, -0.8275, -0.8275],
          [-0.8431, -0.8431, -0.8431,  ..., -0.8431, -0.8431, -0.8431]]],

        [[[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9059, -0.9059, -0.9059,  ..., -0.9059, -0.9059, -0.9059],
          [-0.7804, -0.7804, -0.7804,  ..., -0.7804, -0.7804, -0.7804],
          [-0.7961, -0.7961, -0.7961,  ..., -0.7961, -0.7961, -0.7961]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9451, -0.9451, -0.9451,  ..., -0.9451, -0.9451, -0.9451],
          [-0.8196, -0.8196, -0.8196,  ..., -0.8196, -0.8196, -0.8196],
          [-0.8353, -0.8353, -0.8353,  ..., -0.8353, -0.8353, -0.8353]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9529, -0.9529, -0.9529,  ..., -0.9529, -0.9529, -0.9529],
          [-0.8275, -0.8275, -0.8275,  ..., -0.8275, -0.8275, -0.8275],
          [-0.8431, -0.8431, -0.8431,  ..., -0.8431, -0.8431, -0.8431]]],

        [[[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9059, -0.9059, -0.9059,  ..., -0.9059, -0.9059, -0.9059],
          [-0.7804, -0.7804, -0.7804,  ..., -0.7804, -0.7804, -0.7804],
          [-0.7961, -0.7961, -0.7961,  ..., -0.7961, -0.7961, -0.7961]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9451, -0.9451, -0.9451,  ..., -0.9451, -0.9451, -0.9451],
          [-0.8196, -0.8196, -0.8196,  ..., -0.8196, -0.8196, -0.8196],
          [-0.8353, -0.8353, -0.8353,  ..., -0.8353, -0.8353, -0.8353]],

         [[-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          [-1.0000, -1.0000, -1.0000,  ..., -1.0000, -1.0000, -1.0000],
          ...,
          [-0.9529, -0.9529, -0.9529,  ..., -0.9529, -0.9529, -0.9529],
          [-0.8275, -0.8275, -0.8275,  ..., -0.8275, -0.8275, -0.8275],
          [-0.8431, -0.8431, -0.8431,  ..., -0.8431, -0.8431, -0.8431]]]],
       device='cuda:0'), 'video_grid_thw': tensor([[40, 28, 60]], device='cuda:0', dtype=torch.int32), 'second_per_grid_ts': tensor([0.5192], device='cuda:0')}
generated_ids:tensor([[151644,    872,    198,  ...,  88802,   3837, 104689]],
       device='cuda:0')

进程已结束，退出代码为 0

**Comment by puzhiyuan** [comment]
https://github.com/Kwai-Keye/Keye/issues/12#issuecomment-3047299511
加一个decode的代码
